libraries/ui/src/Banner.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+
+interface BannerProps {
+  title: string;
+  backgroundColor?: string;
+  className?: string;
+  inputPlaceholder?: string;
+  buttonText?: string;
+  showInput?: boolean;
+  showButton?: boolean;
+}
+
+export const Banner: React.FC<BannerProps> = ({
+  title, backgroundColor, className, inputPlaceholder = 'you@example.com', // Default placeholder
+  buttonText = 'Submit', // Default button text
+  showInput = false, // Default to not show input
+  showButton = false,
+}) => {
+  const [inputValue, setInputValue] = useState('');
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    console.log('Submitted value:', inputValue);
+    // TODO: Send the email to the backend
+    setInputValue('');
+  };
+
+  return (
+    <div className={clsx('relative w-full p-5 text-center rounded-lg', className)} style={{ backgroundColor: backgroundColor || '#CCD7FF' }}>
```

Comment:
can we add BEM class style naming? i find this organizational structure really helpful for debugging, testing, etc.
```suggestion
    <div className={clsx('banner relative w-full p-5 text-center rounded-lg', className)} style={{ backgroundColor: backgroundColor || '#CCD7FF' }}>
```